tests/test_media_multipart.py

```diff
@@ -60,7 +60,10 @@
     b'--BOUNDARY\r\n'
     b'Content-Disposition: form-data; name="file"; filename="bytes"\r\n'
     b'Content-Type: application/x-falcon\r\n\r\n'
-    + b'123456789abcdef\n' * 64 * 1024 * 2
+    + b'123456789abcdef\n'
+    * 64
+    * 1024
+    * 2
```

Comment:
this is unfortunate. Maybe adding a `()` helps here?

Reply:
Whoa that was odd indeed. Will try `()`.
OTOH, this is not even the main codebase, but just a unit test file... So I think aesthetics ain't that important here.

Reply:
sure, just notices this ugly part

Reply:
Parentheses do help! Thanks @CaselIT.